src/components/ColorPicker/ColorPicker.scss

```diff
@@ -0,0 +1,364 @@
+@import "../../constants/style";
+
+.color-picker {
+  z-index: 220;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding-bottom: $spacing--xxs;
+  box-shadow: 0 0 20px rgba(55, 62, 79, 0.12);
+  background-color: $gray--300;
+  list-style: none;
+  padding-left: 0;
+  margin-top: auto;
+  border-radius: $rounded--full;
+}
+
+.color-picker > li > button {
+  cursor: pointer;
+  border: none;
+  background: none;
+  padding: 0;
+  width: 40px;
+  height: 40px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &:focus-within {
+    > .column__header-color-option {
+      &--planning-pink {
+        border: 3px solid $pink--100;
+      }
+      &--backlog-blue {
+        border: 3px solid $blue--100;
+      }
+      &--poker-purple {
+        border: 3px solid $purple--100;
+      }
+      &--value-violet {
+        border: 3px solid $violet--100;
+      }
+      &--goal-green {
+        border: 3px solid $green--100;
+      }
+      &--yielding-yellow {
+        border: 3px solid $yellow--100;
+      }
+      &--online-orange {
+        border: 3px solid $orange--100;
+      }
+    }
+  }
+
+  &:focus-visible {
+    border-radius: $rounded--full;
+    outline: none;
+  }
+}
+
+.color-picker > li {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.color-picker > ul > li {
+  padding: 0;
+}
+
+[theme="dark"] {
+  .color-picker {
+    background-color: $navy--600;
+  }
+}
+
+.column__header-color-option {
+  width: 24px;
+  height: 24px;
+  border-radius: $rounded--full;
+
+  &--planning-pink {
+    background-color: $pink--400;
+    border: 3px solid $gray--300;
+
+    &-selected {
+      background-color: $pink--400;
+      border: 3px solid $pink--200;
+    }
+
+    &:hover {
+      background-color: $pink--400;
+      border: 3px solid $pink--100;
+    }
+
+    &:focus-visible {
+      background-color: $pink--400;
+      border: 3px solid $pink--100;
+    }
+  }
+
+  &--backlog-blue {
+    background-color: $blue--400;
+    border: 3px solid $gray--300;
+
+    &-selected {
+      background-color: $blue--400;
+      border: 3px solid $blue--200;
+    }
+
+    &:hover {
+      background-color: $blue--400;
+      border: 3px solid $blue--100;
+    }
+
+    &:focus-visible {
+      background-color: $blue--400;
+      border: 3px solid $blue--100;
+    }
+  }
+
+  &--poker-purple {
+    background-color: $purple--400;
+    border: 3px solid $gray--300;
```

Comment:
like above, you can remove all of the different color definitions with just one by using the accent color